lib/features/mailbox/domain/extensions/list_mailbox_id_extension.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/core/patch_object.dart';
+import 'package:jmap_dart_client/jmap/mail/mailbox/mailbox.dart';
+import 'package:model/mailbox/mailbox_property.dart';
+import 'package:tmail_ui_user/features/mailbox/domain/model/mailbox_subscribe_state.dart';
+
+extension ListMailboxIdExtension on List<MailboxId> {
+
+  Map<Id, PatchObject> generateMapUpdateObjectSubscribeMailbox(MailboxSubscribeState subscribeState) {
+    final Map<Id, PatchObject> maps = {};
+    forEach((mailboxId) {
```

Comment:
why not use `map` instead of `forEach`?

Reply:
You can use different ways to convert `List` to `Map`. Example: 
```
var result = Map.fromIterable(l, key: (v) => v[0], value: (v) => v[1]);
```

But from starting from `Dart 2.3`, we are encouraged to use `collection-for`.
```
var result = { for (var v in l) v[0]: v[1] };
```

So I converted `forEach` to `for-in`. See details in `commit-fixup`